src/Nethermind/Nethermind.Sockets/IWebSocketsModule.cs

```diff
@@ -10,7 +10,7 @@ namespace Nethermind.Sockets
     public interface IWebSocketsModule
     {
         string Name { get; }
-        ISocketsClient CreateClient(WebSocket webSocket, string client, HttpContext context);
+        Task<ISocketsClient> CreateClient(WebSocket webSocket, string client, HttpContext context);
```

Comment:
I would use valuetask as we won't need the task later

Reply:
Or no task at all?

Reply:
One of the interface implementations added an await call. Thats why it was added

Reply:
ah and it is the main one